the-way-we-work.md

```diff
@@ -0,0 +1,664 @@
+- Principals
+- Jacob in particular (e.g. coaching for Senior developers)
+- Senior developers
+- Other developers
+- DMs/POs
+- Anyone else
+
+There are various activities to be distributed amongst the stakeholders:
+
+- Compliance with the process/system itself
+- Collecting information (e.g. to measure the success of our approach)
+- Filtering/evaluating information (e.g. who deserves praise this week)
+- Meetings and interactions (e.g. weekly 1-2-1s, regular discussions with DMs/POs)
+- Promotion/Communication (e.g. about the system, but also public praise for individuals)
+- Decisions that may need to be taken in response to certain triggers
+- Various kinds of support or intervention
+- Production of educational or supporting documentation
+
+### 4.4 Duty of Care    
+
+### 4.5 Finance
+
+
+
+
+
+
+
+**Coaching support**
+
+- First iteration from October to April will be targeted at Senior developers
+- 6 x 1 hour sessions
+- Current and specific issues for each developer will be tackled
+
+There will also be some planned content and exercises covering:
+
+- self-confidence
+- self-knowledge
+- critical thinking skills
+- specific concepts related to agile/lean etc.
+- the specific roles and responsibilities within delivery teams (DM/PO/dev/etc.)
+
+**Reporting**
+
+TODO
+
+What information will we record and pass up the chain. How will that information be used - i.e. why it is important that we are doing this.
+
+
+**Triggers and Interventions**
+
+TODO
+
+Set out in plain language what will be the triggers for intervention.
+
+Also set out what interventions we will be willing to take, including in extreme circumstances.
+
+Example of "routine" intervention:
+
+- E.g. developer fails to complete any of their objectives in a given quarter
+- Discussion to take place with line manager and next manager up to agree what extra support can be provided.
+- Line manager maybe also needs extra support? (Because this shouldn't have happened...)
+
+Example of extreme intervention:
+
+- We may wish to consider offering staff the ability to voluntarily drop down a band if they consistently fail to carry out their expected role or to achieve the objectives they set for themselves.
+- If so, we'd need to do the work now to identify how we could make that work with HR.
+- But we also need a bullet-proof way to fairly (i.e. without bias) establish that this was now the correct outcome, and that we have tried everything else.
+- We'd also need to clearly set out how this is a "kind" intervention for the individual, designed to more appropriately challenge them with work they would be able to succeed at. (Optimum challenge)
+
+Positive triggers:
+
+- Note that crucially we need to set out some triggers for positive actions.
+- For example, if a developer completes an objective then Sean should mention this in the next update he delivers.
+- Plenty of other potential positive triggers: shipping code N sprints in a row, obtaining a new qualification, etc.
```

Comment:
We could show the number of deployments across nhsuk per sprint

Reply:
I'm not sure I follow what you mean with this comment. Can you add more detail? Is it in the right place?

Reply:
Ah sorry. So another positve trigger could be that at the end of the month, we could show how many deployments the teams across nhsuk have completed. This could show us how the deployment frequency is increasing over the year and which teams are deploying what frequency each month.

Reply:
I will incorporate this in a slightly expanded "Reporting" section above